src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, OverlayTrigger, Tooltip } from '@edx/paragon';
+import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
+
+const AssignmentRowActionTableCell = ({ row }) => (
+  <div className="d-flex">
+    {row.original.state === 'allocated' && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
+      >
+        <Icon
+          className="ml-auto mr-0"
+          src={Mail}
+          // eslint-disable-next-line no-console
+          onClick={() => console.log(`Reminding ${row.original.uuid}`)}
+          data-testid={`remind-assignment-${row.original.uuid}`}
+        />
+      </OverlayTrigger>
+    )}
+    <div />
+    <OverlayTrigger
+      key="Cancel"
+      placement="top"
+      overlay={<Tooltip variant="light" id="tooltip-cancel">Cancel assignment</Tooltip>}
+    >
+      <Icon
```

Comment:
[clarification] Pretty sure these are intended to be `IconButton` components, not standalone `Icon` components, thus giving them desired hover/focus states and a large click target:

<img width="676" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/6328c62f-8528-4d54-a1d0-2e3afe68de0d">